app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id="<%= dom_id category %>" class="categories">
+  <img src="<%= category.icon %>" alt="<%= category.name %>">
+  <p>
+    <%= category.name %>
```

Comment:
- [ ] As requested by the previous reviewer, Please ensure you follow the layout designs, typography, and color for all pages, some pages are yet to be styled appropriately 👍🏿

- [ ] Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size and weight.
* Layout: composition and space between elements.

- [ ] Please let's improve the design of our sign-up and log-in pages to look like the provided design 🎨 🖌️ 

| Your design | Original design |
| --- | --- |
| ![Screenshot from 2022-10-30 02-59-45](https://user-images.githubusercontent.com/79058364/198858335-0c43bea6-15d4-4c47-abea-4f63262af6fc.png) | ![Screenshot from 2022-10-30 03-19-50](https://user-images.githubusercontent.com/79058364/198858372-9884a209-36c6-4f90-8fa6-853419b0e323.png)|

- [ ] Kindly make sure your  Category page  looks as the original design 👍🏻 
- [ ] Also make sure your  "add a new category"  button has the same color, background color, padding, font ... from the original design 🎨 👨🏻‍🎨  
- [ ] The "add a new category" must be at the bottom of your page. Kindly make this 👍🏻 

| Your design | Original design |
| --- | --- |
| ![18](https://user-images.githubusercontent.com/79058364/198858322-d4a41c39-8861-4d5c-b170-fe06ee5e2a3c.png) | ![image](https://user-images.githubusercontent.com/79058364/197242090-a472cae4-f81a-41c5-b953-e87a2c3dc6dd.png)|